app/controllers/User/StoreController.js

```diff
@@ -50,13 +50,12 @@ class StoreController {
     }
 
     async invoke(request, response) {
```

Comment:
what for is this controller? we have already RegisterController.
I think this can be removed.